app/ui/components/CustomModal.tsx

```diff
@@ -0,0 +1,53 @@
+import React, {useState} from 'react';
+import {
+  Modal,
+  StyleProp,
+  StyleSheet,
+  Text,
+  View,
+  ViewStyle,
+} from 'react-native';
+import {CustomPressable} from './CustomPressable';
+
+export function CustomModal({
+  style,
+  onClose,
+  children,
+}: {
+  style?: StyleProp<ViewStyle> | undefined;
```

Comment:
```suggestion
  style?: StyleProp<ViewStyle> ;
```

явно вказувати undefined не потрібно, це робить `?`